examples/federated-learning/tff/nvflare/README.md

```diff
@@ -0,0 +1,297 @@
+   ```
+
+1. Push the container image to the Artifact Registry repository:
+
+   ```bash
+   docker image push "${NVFLARE_EXAMPLE_CONTAINER_IMAGE_LOCALIZED_ID_WITH_TAG}"
+   ```
+
+1. Create the ConfigSync folder that will deploy the workload
+
+   ```bash
+   ACM_CONFIG_SYNC_PATH=$(terraform output -raw acm_config_sync_configuration_destination_directory_path)
+   mkdir -p ${ACM_CONFIG_SYNC_PATH}/nvflare/base-storage
+   ```
+
+1. Modify the `kustomization.yaml.tpl` file to add the new name and new tag of the image:
+
+   ```bash
+   envsubst < ../examples/federated-learning/tff/nvflare/templates/kustomization.yaml.tpl > ${ACM_CONFIG_SYNC_PATH}/kustomization.yaml
+   ```
+
+### Create infrastructure to store the models
+
+All the models generated will be stored in a Cloud storage bucket mounted by each pod.
+
+1. Initialize the following Terraform variables:
+
+   ```hcl
+   nvflare = true
+   ```
+
+1. Deploy the infrastructure for NVFlare
+
+   ```bash
+   terraform apply
+   ```
+
+1. Modify the `pv-workspace.yaml.tpl` file to replace the bucket name:
+
+   ```bash
+   NVFLARE_WORKSPACE_BUCKET_NAME="\"$(terraform output -raw nvflare_workspace_bucket_name)\""
+   export NVFLARE_WORKSPACE_BUCKET_NAME
+   envsubst < ../examples/federated-learning/tff/nvflare/templates/pv-workspace.yaml.tpl > ${ACM_CONFIG_SYNC_PATH}/base-storage/pv-workspace.yaml
+   ```
+
+1. Regenerate the `kustomization.yaml` file with all the necessary informations:
+
+   ```bash
+   NVFLARE_EXAMPLE_WORKLOADS_KUBERNETES_NAMESPACE="$(terraform output -raw nvflare_namespace)"
+   export NVFLARE_EXAMPLE_WORKLOADS_KUBERNETES_NAMESPACE
+   envsubst < ../examples/federated-learning/tff/nvflare/templates/pv-workspace.yaml.tpl > ${ACM_CONFIG_SYNC_PATH}/base-storage/pv-workspace.yaml
+   ```
+
+### Create NVFlare folder structure
+
+Now that the persistent volume is created, you can start creating the folder structure and upload it to Cloud Storage.
+
+1. Use the NVFlare image previously built on your workstation:
+
+   ```bash
+   docker run -it ${NVFLARE_EXAMPLE_CONTAINER_IMAGE_LOCALIZED_ID_WITH_TAG} bash
+   ```
+
+1. Run the provisioning tool:
+
+   ```bash
+   cd ${HOME}
+   nvflare provision
+   ```
+
+   You have to choose the non-ha (non high-availability) choice, as ha (high-availability) is not yet supported on Kubernetes. It will create a `project.yml` file that you can customize for your needs.
+
+1. Re-run the provisioning tool inside the container:
+
+   ```bash
+   cd ${HOME}
+   nvflare provision
+   cd ${HOME}/workspace/example_project/prod_00/
+   ```
+
+   The different folders generated represent the infrastructure you will deploy on the reference architecture:
+
+   - `server1` is the server that will aggregate all the results from the computation
+   - `site-1` and `site-2` are the clients that will be connected to the server
+   - `admin@nvidia.com` is the administration client to start and list jobs
+
+### Clone the repository
```

Comment:
Refactor the title because we're not cloning the repository anymore. Maybe `Download nVidia Flare`, or something like that?

Reply:
Done